src/lakefs_spec/__init__.py

```diff
@@ -1 +1 @@
-version = "0.1.0"
+version = "0.1.1"
```

Comment:
Why is this not dynamic? Seems like the ideal opportunity to just grab the version from here in the `pyproject.toml`.

To do this, add 

```
dynamic = ["version"]
```
to the `[project]` block, and 

```
[tool.setuptools.dynamic]
version = { attr = "lakefs_spec.version" }
```
somewhere below.

Reply:
I was going to create a follow-up issue to address version bookkeeping (e.g., with bumpversion), but we can also make the change here.